src/gpodder/plugins/itunes.py

```diff
@@ -39,26 +41,26 @@ def itunes_feed_handler(channel, max_episodes, config):
 
     logger.debug('Detected iTunes feed.')
 
-    itunes_lookup_url = 'https://itunes.apple.com/lookup?entity=podcast&id=' + m.group('podcast_id')
+    itunes_lookup_url = f'https://itunes.apple.com/lookup?entity=podcast&id={m.group("podcast_id")}'
     try:
         json_data = util.read_json(itunes_lookup_url)
 
         if len(json_data['results']) != 1:
-            raise ITunesFeedException('Unsupported number of results: ' + str(len(json_data['results'])))
+            raise ITunesFeedException(f'Unsupported number of results: {str(len(json_data["results"]))}')
```

Comment:
f-strings don't need the string conversion:

```suggestion
            raise ITunesFeedException(f'Unsupported number of results: {len(json_data["results"])}')
```